tasks/run.rb

```diff
@@ -29,6 +29,10 @@ def disabled?(lockfile)
       File.exist?(lockfile)
     end
 
+    def include_report?
+      JSON.parse(STDIN.read)['report']
+    end
```

Comment:
This obfuscates where params are coming from a bit... in the interest of making it easier to add additional params in the future, let's parse params at the beginning of the `run` method. From there, we can pass `params['report']` to the `get_result_from_report` method.